pkg/cri/server/image_pull.go

```diff
@@ -178,18 +187,28 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 			continue
 		}
 		if err := c.createImageReference(ctx, r, image.Target()); err != nil {
+			imagePullingError.WithValues(imageRef).Inc()
 			return nil, fmt.Errorf("failed to create image reference %q: %w", r, err)
 		}
 		// Update image store to reflect the newest state in containerd.
 		// No need to use `updateImage`, because the image reference must
 		// have been managed by the cri plugin.
 		if err := c.imageStore.Update(ctx, r); err != nil {
+			imagePullingError.WithValues(imageRef).Inc()
 			return nil, fmt.Errorf("failed to update image store %q: %w", r, err)
 		}
 	}
 
-	log.G(ctx).Debugf("Pulled image %q with image id %q, repo tag %q, repo digest %q", imageRef, imageID,
-		repoTag, repoDigest)
+	imageNamed, _ := distribution.ParseNamed(ref)
+	domain := distribution.Domain(imageNamed)
+
+	size, _ := image.Size(ctx)
+
+	secondsPulling1MiB := time.Since(startTime).Seconds() / float64(size/1.0/1024/1024)
+	imagePullThoughput.WithValues(domain).Update(time.Duration(secondsPulling1MiB) * time.Second)
```

Comment:
should throughput be "MiBs pulled per second" instead of the other way around?

Reply:
I want to use a pulling speed like you mentioned at first.

However, there is only `LabeledTimer` which is histogram metrics in docker go-metrics.

That's why I make a `secondsPulling1MiB` that is a duration of pulling 1MiB data. 

Do you have any suggestions?